ff_net_worth_calculator/__init__.py

```diff
@@ -104,6 +104,10 @@ def load_alfred_json(url):
 
     data = res.json()
 
+    if 'timestamp' in data or 'version' in data:
+        # this is not a valid alfred.json file
+        return []
```

Comment:
Honestly, i don't like the current form of error-handling. Should be reworked at sometime but for now we can go with that.